src/pages/conference/timeline/data_schedule.tsx

```diff
@@ -0,0 +1,394 @@
+import React from 'react'
```

Comment:
there's no component rendered here, please remove the import and rename the file extension from `tsx` to `ts`